StandardRecord/classes_def.xml

```diff
@@ -86,6 +86,10 @@
   <version ClassVersion="10" checksum="3758140000"/>
  </class>
 
+ <class name="caf::SRArichID"/>
+ 
+ <class name="caf::SRPID"/> 
```

Comment:
need to give these a first version, 10. 
if you have <version ClassVersion="10"/> 
and rebuild, i believe it will auto add the checksum